common/src/unifyfs_configurator.h

```diff
@@ -71,6 +71,7 @@
     UNIFYFS_CFG_CLI(unifyfs, mountpoint, STRING, /unifyfs, "mountpoint directory", NULL, 'm', "specify full path to desired mountpoint") \
     UNIFYFS_CFG(client, max_files, INT, UNIFYFS_MAX_FILES, "client max file count", NULL) \
     UNIFYFS_CFG(client, flatten_writes, BOOL, on, "flatten writes", NULL) \
+    UNIFYFS_CFG(client, local_extents, BOOL, off, "track extents to read back local data", NULL) \
```

Comment:
We should document somewhere (I don't know where's best) about the conditions under which it's safe to enable this option.  We definitely want all users to be aware of it since it's such a huge performance win.

Reply:
Yeah, I agree.  It doesn't feel like this table is the place to do it, since it would get too cluttered.  I think we need another breakout section where we can discuss things in more detail.  @CamStan , do you have an suggestion?

Reply:
Added some more text immediately below the table that lists the option.  This keeps the table concise, but also keeps the limitation close to where the option is listed.  You can see how it shows up here.

https://github.com/LLNL/UnifyFS/blob/clientread/docs/configuration.rst

I tried to also hyperlink the option name to the text where the description lives, but I couldn't tell whether it was really working so removed it.

Reply:
I agree that the best place would be where you put it. If we get more of these and the page looks cluttered, we can rethink. If you want it to stand out a bit more, could always put it in a caution/attention admonition, but probably not necessary.

E.g.:
```
.. Caution::

    Text inside admonition goes here. Indent and put a blank line 
    after to separate from subsequent text
```